TLSharp.Core/DataCenterIPVersion.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace TLSharp.Core
+{
+    /// <summary>
+    /// When the Telegram server responds with a set of addresses to connect to, DataCenterIPVersion indicates a preference 
+    /// for how to choose the IP address to connect to 
+    /// </summary>
+    public enum DataCenterIPVersion
+    {
+        /// <summary>
+        /// Prefers IPv6 addresses if any is passed by Telegram
+        /// </summary>
+        Default = 0,
```

Comment:
why do we need this enum be int-based? there's no need

Reply:
there is no need.

Reply:
cool let's remove the assignments to integers then